File: benchmarks/cpu/divsum/divsum.py
def divsum(n):
    total = 0
    for i in range(1, n + 1):
        s = 0
        for j in range(1, i + 1):
            if i % j == 0:
                s += j
        total += s
    return total

print(divsum(10000))

File: benchmarks/cpu/divsum/divsum.js
function divsum(n) {
    let total = 0;
    for (let i = 1; i <= n; i++) {
        let s = 0;
        for (let j = 1; j <= i; j++) {
            if (i % j === 0) s += j;
        }
        total += s;
    }
    return total;
}

console.log(divsum(10000));
